requirements.txt

```diff
@@ -16,12 +16,12 @@ pybreaker==0.4.4
 pycountry==17.9.23
 pymacaroons==0.12.0
 python-dateutil==2.6.1
-raven[flask]==6.5.0
+raven[flask]==6.10.0
 requests==2.21.0
 requests-cache==0.4.13
 responses==0.9.0
 ruamel.yaml==0.15.72
-talisker[gunicorn]==0.14.3
```

Comment:
`talisker[flask,gunicorn,raven,prometheus]==0.15.0`

Reply:
and you can remove `prometheus_client` dep

Reply:
And in theory I could have removed Werkzeug too, but I think if I remove all those then it really does break things. I'm experimenting again, but could we merge this in a working state first? I'm going to fix up https://github.com/canonical-web-and-design/snapcraft.io/pull/2273 anyway to move to the modern version of things.